bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -1084,17 +1084,17 @@ private Map<Integer, BookieId> getReplacementBookiesByIndexes(
             BookieId bookie = ensemble.get(bookieIndex);
             bookiesToExclude.add(bookie);
         }
-
+        List<BookieId> newEnsemble = new ArrayList<>(ensemble);
```

Comment:
If I understand correctly, this isn't a bug, right? I mean, this hasn't any impact on the existing logic and result, right?

Reply:
Yes, it just makes the log incorrect.

Reply:
Got it